modules/compliance-anatomy.adoc

```diff
@@ -261,12 +286,12 @@ NAME                                                  STATUS   SEVERITY
 rhcos4-e8-worker-audit-rules-dac-modification-chmod   PASS     medium
 ----
 
-// TODO: This shouldn't be a level one in this module
-= Useful labels
+[id="compliance-operator-useful-labels_{context}"]
+== Useful labels
 
-Each pod that is spawned by the compliance-operator is labeled specifically with the scan it belongs to and the work it does. The scan identifier is labeled with the `compliance.openshift.io/scan-name` label. The workload identifier is labeled with the `workload` label.
+Each pod that is spawned by the Compliance Operator is labeled specifically with the scan it belongs to and the work it does. The scan identifier is labeled with the `compliance.openshift.io/scan-name` label. The workload identifier is labeled with the `workload` label.
 
-The compliance-operator schedules the following workloads:
+The Compliance Operator schedules the following workloads:
 
 * *scanner*: Performs the compliance scan.
```

Comment:
I can't comment below but in the `$ oc logs -l workload` command: s/`<container-name>`/`<container_name>`/